plat/imx/common/mxcuart_console.S

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2015-2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <asm_macros.S>
+
+/* UART Control Register Bit Fields.*/
+#define  URXD_CHARRDY    (1 << 15)
```

Comment:
I removed the mxcuart support, the mxcuart is another UART IP rather than LPUART, i.MX8QX/QM are using LPUART ONLY, mxcuart will be added when other i.MX8 platforms added which use it.